guides/source/active_storage_overview.md

```diff
@@ -549,9 +549,9 @@ require a higher level of protection consider implementing
 To generate a permanent URL for a blob, you can pass the blob to the
 [`url_for`][ActionView::RoutingUrlFor#url_for] view helper. This generates a
 URL with the blob's [`signed_id`][ActiveStorage::Blob#signed_id]
-that is routed to the blob's
-[`RedirectController`](ActiveStorage::Blobs::RedirectController).
+that is routed to the blob's [`RedirectController`][ActiveStorage::Blobs::RedirectController]
```

Comment:
@ceritium @p8 Does this also have to be

```
[`RedirectController][`ActiveStorage::Blobs::RedirectController`]
```

There are two anchors for `RedirectController` in this PR.

1. with backticks, and one without
2. should the other anchors be updated?

Reply:
If I understood well, it depends. Since there are two links to `ActiveStorage::Blobs::RedirectController` (lines 552 and 649), we can define two anchors (lines 554 and 689), one per section, or a single one.

If we want to keep two anchors, the first one (552) will work with and without backticks. The second one only works with backticks.

If we want only to keep one anchor, we have to reference them with the backticks to keep the link of 649 simpler.

Do you have any preference? I can address the changes if you like.

Reply:
Ping @zzak @p8, any thoughts?

Reply:
@ceritium I think we should be consistent, can you send a PR?

I remember doing a quick search before and this was the only file that was inconsistently using backticks in anchors.

Reply:
Sure, I will take care as soon as I can.